app/src/main/java/app/MainActivity.kt

```diff
@@ -1,12 +1,114 @@
 package app
 
 import android.os.Bundle
+import android.os.StrictMode
+import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import app.AnimalVilla.R
+import app.AnimalVilla.R.*
+import app.DAO.IPromptDAO
+import app.DTO.Prompt
+import org.json.JSONArray
+import org.json.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.URL
+import java.util.concurrent.LinkedBlockingQueue
+import kotlin.concurrent.thread
+
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(layout.activity_main)
+        findViewById<Button>(id.StartButton).setOnClickListener{
+            println("button clicked")
+            //Thread {
+              //  runOnUiThread {
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.